src/main/kotlin/de/bund/digitalservice/useid/transactioninfo/TransactionInfoController.kt

```diff
@@ -0,0 +1,68 @@
+package de.bund.digitalservice.useid.transactioninfo
+
+import io.swagger.v3.oas.annotations.tags.Tag
+import mu.KotlinLogging
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+import java.util.UUID
+
+internal const val TRANSACTION_INFO_SUFFIX = "transaction-info"
+
+@RestController
+@Tag(name = "Transaction Info", description = "Additional information regarding the identification which will be displayed in the eID-Client.")
+@ConditionalOnProperty(name = ["features.desktop-solution-enabled"], havingValue = "true")
+class TransactionInfoController(
+    private val transactionInfoService: TransactionInfoService
+) {
+    private val log = KotlinLogging.logger {}
+
+    @PostMapping(
+        path = ["/api/v1/identification/sessions/{useIDSessionId}/$TRANSACTION_INFO_SUFFIX"],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    fun createTransactionInfo(
+        @PathVariable useIDSessionId: UUID,
+        @RequestBody transactionInfo: TransactionInfo,
+        authentication: Authentication
+    ): Mono<ResponseEntity<TransactionInfo>> {
+        return transactionInfoService.create(useIDSessionId, transactionInfo)
```

Comment:
Perspektivisch holen wir Name und URL aus der Umgebung, oder? (Deshalb auch `authentication` als Parameter?) Für den Prototypen aber erstmal egal.

Reply:
Der `authentication` Parameter ist ein Überbleibsel vom copy and paste. Entferne ich. 
Zukünftig würde es wahrscheinlich Sinn machen, Namen und URL pro Kunde fest zu konfigurieren, ja. Für den Prototypen würde ich es erstmal so flexibel lassen.